apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -634,7 +675,7 @@ wt_type_e _wt_get_opt(int argc, char *argv[])
 	if (argc < WT_MIN_ARG) {
 		return WT_TYPE_ERR;
 	}
-
+	
```

Comment:
you can remove extra spaces in source code. Please check at all places

Reply:
Done